4337/README.md

```diff
@@ -82,7 +82,7 @@ bytes memory deployData = abi.encodeWithSignature("createProxyWithNonce", SAFE_S
 bytes memory initCode = abi.encodePacked(SAFE_PROXY_FACTORY_ADDRESS, deployData);
 ```
 
-The diagram below outlines the flow that is triggered by the initialization data to deploy a Safe with the 4337 module enabled.
+The diagram below outlines the flow triggered by the initialization data to deploy a Safe with the 4337 module enabled.
```

Comment:
I realised that there is a bug in the sequence diagram 👇 that I missed in #112 that you can ninja edit in this PR: the 4337 is set as the fallback handler **and** enabled as a module (the `EntryPoint` is no longer enabled as a module).